lib/config/index.js

```diff
@@ -80,6 +80,15 @@ if (!(config.defaultPermission in config.permission)) {
   config.defaultPermission = config.permission.editable
 }
 
+if (!(config.lockGuestNotes.permission in config.permission)) {
+  logger.error('Invalid lockGuestNotes permission: %s\nAvailable permissions are: %s', config.lockGuestNotes, Object.keys(config.permission).toString())
+}
+
+if (config.lockGuestNotes.timeout < 0) {
```

Comment:
Is there some place where this is already coerced into a number? What would happen if I set `timeout` to `"tuesday"`?
If not, this needs a `parseInt(..., 10);`.

Reply:
Funnily enough inverting the expression is enough: `(!(config.lockGuestNotes.timeout >= 0))`